EIPS/rentcollector.md

```diff
@@ -0,0 +1,73 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Rent Collection for Blockchain Data
+    Author: Daniel A. Nagy <daniel@ethereum.org>, Zsolt Felfoldi <zsolt@ethereum.org>
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 2017-11-01
+
+
+## Simple Summary
+Introducing a way to collect rent from accounts that is proportional 
+to the cost that they impose on the network by storing a large state
+over a long period of time.
+
+## Abstract
+A new global parameter called `rent` similar to `gasLimit` that can be 
+modified by miner vote is introduced. It is initialized at 0 and is 
+assumed to have been zero before its introduction. The miner validating 
+a particular block can increase rent in a bounded way at the cost of 
+forfeiting a proportional part of the mining reward. Similarly, if 
+`rent` is more than zero, one can decrease it in a bounded way and 
+receive the same additional mining reward that would need to be 
+forfeited to raise it back.
+
+Each contract account gets a new balance called `lease` used for paying 
```

Comment:
Why have a separate `lease` balance rather than just using `eth` balance?  Should the lease balance run out and the contract is pruned it seems wasteful to have any ETH the contract was holding suddenly be black holed when that ETH could have gone to paying rent.  Similarly, adding a separate source of ETH adds complexity and I don't see any advantage to separating the balance.

Reply:
Agreed. The `lease` balance is redundant.